temporal-sdk/src/main/java/io/temporal/internal/sync/ActivityInfoImpl.java

```diff
@@ -134,4 +135,8 @@ public boolean isLocal() {
     }
     return Optional.empty();
   }
+
+  public Header getHeader() {
```

Comment:
I think this should also be added into `ActivityInfo` interface.

Reply:
I don't think we want to expose the Header protobuf to the activity code directly.

Reply:
There is already a couple of methods like this (getInput and getCompletionHandle) - what do you guys think about introducing an interface like ActivityInfoInternal where we can define these methods and wouldn't allow this interface to escape TemporalSdk internals?